src/Symfony/Component/DependencyInjection/Loader/Configurator/InlineServiceConfigurator.php

```diff
@@ -29,6 +29,8 @@ class InlineServiceConfigurator extends AbstractConfigurator
     use Traits\ParentTrait;
     use Traits\TagTrait;
 
+    private $definition;
```

Comment:
@nicolas-grekas: Perhaps this was intended somehow? But anyway in 
https://github.com/symfony/symfony/blob/e3b000d1b3b6b02944839ac48eadd3c93380f52f/src/Symfony/Component/DependencyInjection/Loader/Configurator/AbstractConfigurator.php#L65-L75

 you're already rebinding to access it.

Reply:
good catch, I forgot declaring the private property. Not sure about protected: would you move it to AbstractConfigurator? If it makes sense, you can give it a try yes (I don't remember and didn't look in more details :) )

Reply:
I guess you didn't create this property in `AbstractConfigurator` because not all children of this class use a definition (on contrary of `AbstractServiceConfigurator`.
I was thinking about abusing a bit OOP: https://3v4l.org/cgGcp

Reply:
Ok

Reply:
Done.